test_settings.py

```diff
@@ -36,5 +36,6 @@
 # DRF Settings
 REST_FRAMEWORK = {
     'PAGE_SIZE': 20,
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
```

Comment:
@doctoryes turns out this was the thing breaking the test.  DRF 3.7 removed the default pagination class (and clearly labeled the change in the release notes...).

Reply:
Good find! And good to know. I feel like DRF could have deprecated/removed it in a more informative way - but fine.